client/go.mod

```diff
@@ -31,11 +31,11 @@ require (
 	github.com/pkg/errors v0.9.1 // indirect
 	github.com/spf13/pflag v1.0.5 // indirect
 	golang.org/x/mod v0.6.0 // indirect
-	golang.org/x/net v0.1.1-0.20221027164007-c63010009c80 // indirect
+	golang.org/x/net v0.3.1-0.20221206200815-1e63c2f08a10 // indirect
```

Comment:
@sunnylovestiramisu Can you submit another PR to make a similar change in client/go.mod as follows:
replace golang.org/x/net => golang.org/x/net v0.4.0

I'm getting a dependencies change error when generating the changelog.  I wonder if it is because the client and server go.mod are having different libs.